CONTRIBUTING.md

```diff
@@ -1,19 +1,5 @@
 # Contributing to rhc playbook verifier
 
-## Security
```

Comment:
There's already a section in the readme detailing how to test and lint the code. I'd prefer for the docs to be DRY.